libs/agno/agno/knowledge/reader/reader_factory.py

```diff
@@ -238,13 +238,25 @@ def get_reader_for_url_file(cls, extension: str) -> Reader:
 
     @classmethod
     def get_all_reader_keys(cls) -> List[str]:
-        """Get all available reader keys."""
+        """Get all available reader keys in priority order."""
         # Extract reader keys from method names
         reader_keys = []
         for attr_name in dir(cls):
             if attr_name.startswith("_get_") and attr_name.endswith("_reader"):
                 reader_key = attr_name[5:-7]  # Remove "_get_" prefix and "_reader" suffix
                 reader_keys.append(reader_key)
```

Comment:
```suggestion
    PREFIX = "_get_"
    SUFFIX = "_reader"
    
    reader_keys = []
    for attr_name in dir(cls):
        if attr_name.startswith(PREFIX) and attr_name.endswith(SUFFIX):
            reader_key = attr_name[len(PREFIX):-len(SUFFIX)]
            reader_keys.append(reader_key)
```

Clarifies that `[5:-7]` part